<?xml version="1.0" encoding="UTF-8"?>
<!--
Copyright © 2014 Michael Jung

This file is part of Panga.

Panga is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

Panga is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with Panga.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>Chi2Explorer</class>
 <widget class="QMainWindow" name="Chi2Explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>χ² Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <item>
     <widget class="ContourPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="sample_horizontal_layout">
        <item>
         <widget class="QLabel" name="sample_label">
          <property name="text">
           <string>Sample:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sample_combobox"/>
        </item>
        <item>
         <spacer name="upper_horizontal_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="parameter_config_tableview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outer_slider_layout">
        <item>
         <spacer name="left_slider_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="inner_slider_layout"/>
        </item>
        <item>
         <spacer name="right_slider_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="right_vertical_spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="chi2_scale_groupbox">
        <property name="title">
         <string>χ² scale</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QCheckBox" name="chi2_scale_autoupdate_checkbox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="chi2_scale_minimum_lineedit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="chi2_scale_maximum_lineedit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="chi2_scale_minimum_label">
           <property name="text">
            <string>Minimum:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="chi2_scale_maximum_label">
           <property name="text">
            <string>Maximum:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="chi2_scale_autoupdate_label">
           <property name="text">
            <string>Auto-update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottom_right_horizontal_layout">
        <item>
         <widget class="QPushButton" name="update_plot_button">
          <property name="text">
           <string>Update plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoupdate_plot_checkbox">
          <property name="text">
           <string>auto-update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContourPlot</class>
   <extends>QWidget</extends>
   <header>contourplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
